src/utils/cloning.ts

```diff
@@ -410,13 +411,16 @@ function cloneASTNodeImpl<T extends ASTNode>(
       [...node.vOverrides].map((o) => cloneASTNodeImpl(o, ast, remappedIds)),
       node.raw,
     );
+  } else if (node instanceof PlaceholderStatement) {
+    newNode = clonePlaceholder(node, ast, remappedIds);
   }
 
   if (notNull(newNode) && sameType(newNode, node)) {
     ast.setContextRecursive(newNode);
     ast.copyRegisteredImports(node, newNode);
     return newNode;
   } else {
+    console.log(node);
```

Comment:
Is this print still needed, given that the error below includes which node was not cloneable?